### PR TITLE
Assorted fixes

### PR DESCRIPTION
**Pull request by dagwieers**
This PR includes:
- Remove items from .gitattributes
- Remove WHITEIP from ARM HWID list (it's Intel Apollo Lake)
- Fix wrong URL in setup.py

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,10 +1,8 @@
 .github/ export-ignore
-dist/ export-ignore
 tests/ export-ignore
 .gitattributes export-ignore
 .gitignore export-ignore
 .pylintrc export-ignore
-codecov.yml export-ignore
```
Comment (mediaminister) [inline .gitattributes:7]: I don't understand, does this mean `codecov.yml` should be exported now in zip packages?

Comment (michaelarnauts) [reply .gitattributes:7]: It think the file is removed from the repo, and the settings are defined on the organisation.

Comment (dagwieers) [reply .gitattributes:7]: Yes, I forgot to clean it up earlier.

Having codecov defined  at the org lrvel has one benefit, it doesn't suddenly fail to work. But if there is a need to have to tweak it, we can still reintroduce it.

Comment (mediaminister) [reply .gitattributes:7]: There is still a codecov.yml in this repo!

**Comment by dagwieers** [comment]
:roll_eyes: Fixed.

**Pull request merged**